docs/manifest-stage.json

```diff
@@ -0,0 +1,138 @@
+{
+"name": "Adobe Stock",
+"version": "1.0.1",
+"description": "Adobe Stock",
+"author": "Rachel Luxemburg",
```

Comment:
I'm not sure that's the correct value here, we probably want to list the Adobe Stock Team as the author?

Reply:
ok, thanks for the feedback. Let's stick to the general practice then.